src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -178,7 +178,18 @@ def to_content_str_or_list(chat_str: str, hash2images: Mapping):
     return result if include_image else chat_str
 
 
-def handle_openai_error(tries: int = 10, delay: float = 8.0):
+def generate_retry_interval(retry_count: int) -> float:
+    minBackoffInSec = 3
```

Comment:
better to use snake case for variable name?

Reply:
changed.